FileShares/src/FileSharesWeb/Models/ErrorViewModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace Steeltoe.Samples.FileSharesWeb.Models;
+
+public sealed class ErrorViewModel
+{
+    public string? RequestId { get; init; }
```

Comment:
This should not have been changed to `init`, it is boilerplate ASP.NET code.

Reply:
Where do we draw the line on what stays from the template and what should be changed? There are probably at least a half dozen resharper-induced changes to template files for every sample (eg: sealing this class)

Reply:
I don't have a good answer for that right now. In general, I think we shouldn't change boilerplate code unless there's a good reason to do so.